### DISCUSSION TITLE
How do i get galaxy brain?

### DISCUSSION THREAD

**Discussion by Neon-King** [community/community#19746] [Other Feature Feedback, Questions, & Ideas] [answered]
what are the req for it?

**Comment by Drix10** [comment, marked as answer]
You need to Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.
Hope this answer has helped you, and all the best in getting the achievement! 🙂

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @Neon-King. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Pavel-Shcherbo** [comment]
How do i get galaxy?

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
